### ISSUE TITLE
Text color in active cell, dark mode

### ISSUE THREAD

**Issue by ttowntom** [NoroffFEU/yeetsheet#88]
Due to accessibility reasons the text color of active text in dark mode should be changed to uphold WCAG compatability.
